cloudformation/ccf-athena.yaml

```
@@ -1,3 +1,16 @@
+Parameters:
+  AuthorizedRole:
+    Description: The trusted entity's ARN that can assume the CCF role
+    Type: String
+    Default: arn:aws:iam::<ACCOUNT ID>:<NAME>
```

I think we can do something like this here

```suggestion
    Default: !Sub arn:aws:iam::${AWS::AccountId}:role/<ROLE_NAME>
```

---

Good tip! Didn't know that placeholder. But we can have the case where the authorized role or user doesn't live in the same account that the ccf-athena role. So that may be confusing